mobileapp/ios/App/App.xcodeproj/project.pbxproj

```diff
@@ -194,10 +194,10 @@
 			buildActionMask = 2147483647;
 			files = (
 			);
-			inputPaths = (
+			inputFileListPaths = (
 			);
 			name = "[CP] Embed Pods Frameworks";
-			outputPaths = (
+			outputFileListPaths = (
```

Comment:
I assume this is just a update to use a new syntax/config?

Reply:
Not 100% what caused this but it was a generated change during the capacitor build process

Reply:
Let's not mess with the generated files 😅